File: Python/Minor_Assignment_3/Q17.py
def vowelRemove(s):
    for i in ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']:
        s = s.replace(i, '')
    return s

s=input("Enter a string: ")
print("After removing vowels:",vowelRemove(s))


'''
Output:
Enter a string: anish
After removing vowels: nsh
'''

File: PPWC/Minor_Assignment_5/Q17.c
#include <stdio.h>
int main(){
    int b=65;
    void *p=&b;
    int *j=(int *)p;
    char *ch=(char *)p;
    printf("%d %c\n",*j,*ch);
    return 0;
}

/*
Output--
65 A
*/